src/components/CheckForm/FormLayout/FormLayout.test.tsx

```diff
@@ -77,7 +77,7 @@ describe(`FormLayout`, () => {
         <FormLayout.Section label="First section">
           <div>{firstSectionText}</div>
         </FormLayout.Section>
-        <FormLayout.Section label="Second section" isOpen>
+        <FormLayout.Section label="Second section">
```

Comment:
Tests still need to be updated, didn't want to fiddle with them until I know we've got consensus on the direction though